collector.go

```diff
@@ -74,8 +74,7 @@ func (c *DockerCollector) processContainer(container types.Container, ch chan<-
 		if stats, err := c.cli.ContainerStats(context.Background(), container.ID, false); err != nil {
 			log.Fatal(err)
 		} else {
-
-			var containerStats containerStats
+			var containerStats types.StatsJSON
```

Comment:
👍 Didn't know that there is already a Go struct exist, was seeking last time but did't found. This is much better, since the lib maintainer will keep all API changes in sync